CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+## 1.5.0-beta.1
+
+## ✨ Features
+* GeolocateControl displays a `disabled` icon if user denies geolocation permission. [#8871](https://github.com/mapbox/mapbox-gl-js/pull/8871))
+* Add `mapboxgl.getRTLTextPluginStatus()` to query the current status of the `rtl-text-plugin` ([#8864](https://github.com/mapbox/mapbox-gl-js/pull/8864))
```

Comment:
what benefit does this provide for the customer? what can I do now with the RTL text plugin that I couldn't do before?

Reply:
Rephrased, and referenced the issue as well.